controllers/TrendingArticlesController.js

```diff
@@ -18,33 +20,67 @@ export default class Trending {
      *
      * @param {object} req request object
      * @param {object} res response object
+     * @param {function} next function
      *
      * @returns {object} trending data
      * @memberof Trending
      */
-  static async trend(req, res) {
-    const itsTrending = await Article.findAll({
-      limit: 10,
-      where: {
-        status: 'published'
-      },
-      include: [
-        { // Author
-          model: User,
-          as: 'author',
-          attributes: ['id', 'firstName', 'lastName', 'email'],
-          include: [{
-            model: Profile,
-            as: 'profile',
-          }]
+  static async trend(req, res, next) {
+    try {
+      const { tag } = req.query;
+      if (tag) {
+        const itsTrending = await Article.findAll({
+          limit: 10,
+          where: {
+            status: 'published',
+            tagsList: {
+              [Op.contains]: [`${tag}`],
+            },
+          },
+          include: [
+            { // Author
+              model: User,
+              as: 'author',
+              attributes: ['id', 'firstName', 'lastName', 'email'],
+              include: [{
+                model: Profile,
+                as: 'profile',
+              }]
+            },
+          ],
+          order: [
```

Comment:
Hey Tolu, is it possible to use one array when ordering these properties?

Reply:
This is how you order records in Sequelize https://sequelize.org/master/manual/querying.html#where